Include/internal/pycore_compile.h

```diff
@@ -40,6 +40,13 @@ extern int _PyAST_Optimize(
     _PyASTOptimizeState *state);
 
 /* Access compiler internals for unit testing */
+
+PyAPI_FUNC(PyObject*) _PyCompile_CodeGen(
```

Comment:
Does this (or the other functions) need to be an API function?

Reply:
Can _testinternalcapi import them otherwise?

Reply:
I think so, that why it has "internal" in the name.
It defines `Py_BUILD_CORE_BUILTIN`

Reply:
It doesn't seem to work:

`[ERROR] _testinternalcapi failed to import: dlopen(/Users/iritkatriel/src/cpython-654/build/lib.macosx-13.0-x86_64-3.12-pydebug/_testinternalcapi.cpython-312d-darwin.so, 0x0002): symbol not found in flat namespace '__PyCompile_CodeGen'`